pkg/sdkserver/sdkserver.go

```diff
@@ -263,6 +267,38 @@ func (s *SDKServer) Run(ctx context.Context) error {
 	return nil
 }
 
+// waitForConnection attempts a GameServer GET every 3s until the client responds.
```

Comment:
Amusingly this matches the exact pattern we use on SDK clients that don't support gRPC.